docs/cli-tool.md

```diff
@@ -36,8 +36,13 @@ following block in the config:
 insecure: true
 ```
 
-Alternatively, both of these values could be provided via CLI flags `--server`
-and `--insecure`.
+JWKS endpoint is also required if you want to validate justification tokens, it will default to the server domain if it is not specified.
```

Comment:
```suggestion
JWKS endpoint is also required if you want to validate justification tokens, it will default to the server domain + `/.well-known/jwks` if it is not specified.
```